src/supernode/DAPI_RPC_Server.cpp

```diff
@@ -104,27 +95,21 @@ bool supernode::DAPI_RPC_Server::HandleRequest(const epee::net_utils::http::http
     SCallHandler* handler = nullptr;
 
     {
-    	//LOG_PRINT_L5("\n\n\n");
-
     	boost::lock_guard<boost::recursive_mutex> lock(m_Handlers_Guard);
     	for(unsigned i=0;i<m_vHandlers.size();i++) {
     		SHandlerData& hh = m_vHandlers[i];
 
-    		//LOG_PRINT_L5("Have: '"<<hh.Name<<"' : '"<<hh.PaymentID<<"' need: '"<<callback_name<<"' : '"<<payment_id<<"'");
-
     		if(hh.Name!=callback_name) continue;
     		if(hh.PaymentID.size() &&  hh.PaymentID!=payment_id) continue;
 
     		handler = hh.Handler;
     		break;
     	}
-    	//LOG_PRINT_L5("\n\n\n");
     }
+    LOG_PRINT_L2(response_info.m_body);
 
-    if(!handler) { LOG_PRINT_L5("handler not found for: "<<callback_name); return false; }
-    //LOG_PRINT_L5("Befor process: "<<callback_name<<"  in: "<<Port());
-    if( !handler->Process(ps, response_info.m_body) ) { LOG_PRINT_L5("Fail to process (ret false): "<<callback_name); return false; }
-    //LOG_PRINT_L5("After process: "<<callback_name<<"  in: "<<Port());
+    if(!handler) { LOG_PRINT_L4("handler not found for: "<<callback_name); return false; }
```

Comment:
Maybe it's better to log errors to LOG_ERROR?

Reply:
Fixed